community/php-xdebug/APKBUILD

```diff
@@ -0,0 +1,68 @@
+# Contributor: Andy Postnikov <apostnikov@gmail.com>
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Contributor: Fabio Ribeiro <fabiorphp@gmail.com>
+# Maintainer: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Fabio Ribeiro <fabiorphp@gmail.com>
+# Maintainer: Andy Postnikov <apostnikov@gmail.com>
+pkgname=php-xdebug
+_pkgname=xdebug
+pkgver=2.4.1
+pkgrel=0
+pkgdesc="PHP extension provides functions for function traces and profiling"
+url="http://pecl.php.net/package/$_pkgname"
+arch="noarch"
+license="PHP"
+depends=""
+makedepends="autoconf"
+install=""
+subpackages=""
+source="http://pecl.php.net/get/$_pkgname-$pkgver.tgz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+_phpversions="5 7"
+for _v in $_phpversions; do
+	makedepends="$makedepends php$_v-dev"
+	subpackages="$subpackages php$_v-${pkgname#php-}:_subpackage"
+done
+
+prepare() {
+	default_prepare || return 1
+
+	local ver; for ver in $_phpversions; do
+		cp -ar "$builddir" "$builddir-$ver"
+	done
+}
+
+build() {
+	local ver; for ver in $_phpversions; do
+		cd "$builddir-$ver"
+
+		phpize$ver || return 1
+		./configure --prefix=/usr \
+			--with-php-config=/usr/bin/php-config$ver || return 1
+		make || return 1
+		make test || return 1
+	done
+}
+
+package() {
+	mkdir -p "$pkgdir"
+}
+
+_subpackage() {
+	local ver="${subpkgname:3:1}"
+	pkgdesc="$pkgdesc (for PHP $ver)"
+	arch="all"
+	depends="$depends php$ver"
+	install_if="$pkgname=$pkgver-r$pkgrel php$ver"
+	[ "$ver" = 5 ] && replaces="$pkgname"
+
+	make -C "$builddir-$ver" INSTALL_ROOT="$subpkgdir" install || return 1
+
+	install -d "$subpkgdir"/etc/php$ver/conf.d || return 1
+	echo "extension=$_pkgname.so" > "$subpkgdir"/etc/php$ver/conf.d/$_pkgname.ini
```

Comment:
This probably should be removed to prevent enabling the extension by default because it affects performance and require manual configuration in 90% of cases.
But this breaks backward compatibility...